security/design/security.go

```diff
@@ -172,7 +172,7 @@ func (s *SchemeExpr) Type() string {
 	case JWTKind:
 		return "JWT"
 	default:
-		return "[unknown]"
+		panic(fmt.Sprintf("unknown scheme kind: %#v", s.Kind)) // bug
```

Comment:
I think there are places in the code where we check whether the type is `[unknown]` - these checks are not needed anymore either correct?